catalyst_voices/apps/voices/lib/pages/registration/wallet_link/stage/select_wallet_panel.dart

```diff
@@ -76,6 +79,13 @@ class _SelectWalletPanelState extends State<SelectWalletPanel> {
       registration.nextStep();
     }
   }
+
+  Future<void> _launchSupportedWalletsLink() async {
+    final url = VoicesConstants.supportedWalletsUrl.getUri();
+    if (!await launchUrl(url)) {
+      throw Exception('Could not launch $url');
```

Comment:
Should be here created custom exception extended from LocalizedException?

Reply:
It's very unlikely that this fails. LocalizedExceptions should be used for signaling to the user known and expected exceptions. When running the app in a browser this will always work, when running in mobile it will only fail if there's no browser installed (very unlikely). I would live it as it is or even don't throw the exception at all

Reply:
Leaving the exception here might be a good idea, in case the url is broken, etc

Reply:
Okey. Thanks for explaining the usage of LocalizedException.